app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/GoodsItemRecords.scala

```diff
@@ -139,7 +139,7 @@ object GoodsItemRecords {
       version = eisGoodsItemRecords.version,
       active = eisGoodsItemRecords.active,
       toReview = eisGoodsItemRecords.toReview,
-      reviewReason = eisGoodsItemRecords.reviewReason,
+      reviewReason = translateReviewReason(eisGoodsItemRecords.reviewReason, eisGoodsItemRecords.toReview),
```

Comment:
you may have to do the translation in `CreateOrUpdateRecordResponse.scala` as well where we are returning reviewReason field.

Reply:
we don't need to make any changes in CreateOrUpdateRecordResponse.scala